sn_interface/Cargo.toml

```
@@ -30,18 +30,18 @@ itertools = "~0.10.0"
 lazy_static = "1"
 multibase = "~0.9.1"
 num_cpus = "1.13.0"
-proptest = { version ="1.0.0", optional =true }
-qp2p = "0.36.1"
+proptest = { version = "1.0.0", optional = true }
+qp2p = "~0.36.1"
 rand = "~0.8.5"
-rand-07 = { package = "rand", version = "0.7.3" } # required till ed25519-dalek upgrades to rand v0.8
+rand-07 = { package = "rand", version = "0.7.3" }                                    # required till ed25519-dalek upgrades to rand v0.8
 rayon = "1.5.1"
 rmp-serde = "1.0.0"
 self_encryption = "~0.27.5"
 serde = { version = "1.0.111", features = ["derive", "rc"] }
 serde_bytes = "~0.11.5"
 serde_json = "1.0.53"
 signature = "1.1.10"
-sn_consensus = "3.2.1"
+sn_consensus = { version = "3.2", git = "https://github.com/maidsafe/sn_consensus" }
```

let's drop the git reference now that sn_consensus is published.
```suggestion
sn_consensus = "3.2.1"
```